sdk/monitor/monitor-opentelemetry/README.md

```diff
@@ -79,6 +79,11 @@ const options: AzureMonitorOpenTelemetryOptions = {
         redis: { enabled: true },
         redis4: { enabled: true },
     },
+    browserSdkLoaderOptions: {
+        enableBrowserSdkLoader: false,
```

Comment:
remove the broswerSDK string in all options as all of them are already part of something called browserSdkLoaderOptions